src/Extensions/Abstractions/EventSourcesEventIds.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	/// <summary>
+	/// Event ids enum
+	/// </summary>
+	public enum EventSourcesEventIds
+	{
+		/// <summary>
+		/// Event Id for logging general info message
+		/// </summary>
+		LogInfoEventId = 1,
+
+
+		/// <summary>
+		/// Event Id for logging error message
+		/// </summary>
+		LogErrorEventId = 2,
```

Comment:
Is there a need to maintain a specific ordering? If not, I'd suggest the hierarchy should be maintained so that for instance Spam and Verbose come before Error, etc. If the ordering does need to be maintained, I'd suggest putting this in a comment so that someone doesn't "correct" it

Reply:
Usually, in enums with corresponding numbers are ordered by ascending and I don't think we should change it, although I agree that log events should've been ordered initially.
We can attempt to rename them after verifying that current state working with our log aggregation.